crates/tools/tests/cli.rs

```diff
@@ -0,0 +1,20 @@
+extern crate tools;
+
+use std::path::Path;
+use tools::{render_template, update};
+
+const GRAMMAR: &str = "../ra_syntax/src/grammar.ron";
+const SYNTAX_KINDS: &str = "../ra_syntax/src/syntax_kinds/generated.rs";
+const SYNTAX_KINDS_TEMPLATE: &str = "../ra_syntax/src/syntax_kinds/generated.rs.tera";
+const AST: &str = "../ra_syntax/src/ast/generated.rs";
+const AST_TEMPLATE: &str = "../ra_syntax/src/ast/generated.rs.tera";
```

Comment:
I think we should try to avoid duplicating these paths across main.rs/cli.rs.

Let's move them to `lib.rs`, and use `CARGO_MANIFEST_DIR` to generate full paths correctly regradless of the binary working directory?